migrations/20221011041400_add_mfa_indexes.up.sql

```diff
@@ -0,0 +1,9 @@
+create index if not exists refresh_token_session_id on {{ index .Options "Namespace" }}.refresh_tokens using btree(session_id);
+alter table {{ index .Options "Namespace" }}.mfa_amr_claims
+add column if not exists id uuid not null,
+add constraint amr_id_pk primary key(id);
+
```

Comment:
Some better spacing to improve readability?

```suggestion

alter table {{ index .Options "Namespace" }}.mfa_amr_claims
  add column if not exists id uuid not null,
  add constraint amr_id_pk primary key(id);
```